src/imeta.cpp

```diff
@@ -2452,12 +2452,15 @@ usage( const char *subOpt ) {
         }
         if ( strcmp( subOpt, "qu" ) == 0 ) {
             char *msgs[] = {
-                " qu -d|C|R|u AttName Op AttVal [...] (Query objects with matching AVUs)",
+                " qu -d|C|R|u AttName Operator AttVal [...] (Query objects with matching AVUs)",
                 "Query across AVUs for the specified type of item",
                 "Example: qu -d distance '<=' 12",
                 " ",
+                "The Operator can be any of the following:",
+                " <, >, <=, >= <>, =, !=, like, not like",
```

Comment:
Should there be a comma between >= and <>?

Reply:
done